pkgs/servers/headscale/default.nix

```diff
@@ -6,16 +6,16 @@
 }:
 buildGoModule rec {
   pname = "headscale";
-  version = "0.21.0";
+  version = "0.22.1";
 
   src = fetchFromGitHub {
     owner = "juanfont";
     repo = "headscale";
     rev = "v${version}";
-    hash = "sha256-Y4fTCEKK7iRbfijQAvYgXWVa/6TlPikXnqyBI8b990s=";
+    sha256 = "sha256-6T4wWuhikanoQGGjVvNJak5yvgcEfhGtOmfLc2xKmms=";
   };
 
-  vendorHash = "sha256-R183PDeAUnNwNV8iE3b22S5hGPJG8aZQGdENGqcPCw8=";
+  vendorSha256 = "sha256-+JxS4Q6rTpdBwms2nkVDY/Kluv2qu2T0BaOIjfeX85M=";
```

Comment:
These should have stayed hash and vendorHash

Reply:
Deal. I'll put up a PR to revert the attributes. Is `*sha256` legacy compared to `*hash` such that both are supported?

Reply:
Yeah SRI identified hashes have been growing in use in nixpkgs. Youd have to be on quite an old copy of nix to not support them AFAIK

Reply:
@06kellyjac I pushed a PR to fix this but it was commented, there, that it's still either-or, for now: https://github.com/NixOS/nixpkgs/pull/227783/commits/84be28f36ac4ce0632df8ff2cd91822b234360db .